sdk/communication/communication-administration/samples/javascript/package.json

```diff
@@ -24,6 +24,7 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/communication-administration": "latest",
+    "@azure/communication-sms": "latest",
```

Comment:
It's a nice idea but I don't think we should cross-ref another service client like Sms but stay self-contained.

Unless @xirzec thinks it's okay.

Reply:
If they're really tied together, then it would be okay, but we always try to keep samples within the package folders very elemental. They should show how to accomplish fundamental tasks using the package. Samples of more integrated or broader design themes using the service usually would go in a different repository in the Azure-Samples org. Up to you all to decide what those fundamental tasks are.